pkg/startup/v6/data/master/etc/origin/node/node-config.yaml

```diff
@@ -33,6 +33,14 @@ kubeletArguments:
   - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
   max-pods:
   - "50"
+  image-gc-low-threshold: 
+  - "60"
+  maximum-dead-containers:
+  - '50'
+  maximum-dead-containers-per-container:
+  - '2'
+  minimum-container-ttl-duration:
+  - 5m
```

Comment:
Upstream defaults ([doc link](https://kubernetes.io/docs/concepts/cluster-administration/kubelet-garbage-collection/#user-configuration)):

- `image-gc-high-threshold` the percent of disk usage which triggers image garbage collection (85%) **<-- we're not modifying this but maybe we should?**
- `image-gc-low-threshold` the percent of disk usage to which image garbage collection attempts to free (80%)
- `minimum-container-ttl-duration` minimum age for a finished container before it is garbage collected. Default is 0 minute, which means every finished container will be garbage collected **<-- we seem to be increasing the minimum ttl here?**
- `maximum-dead-containers-per-container` maximum number of old instances to be retained per container. Default is 1. **<-- also increasing this?**
- `maximum-dead-containers` maximum number of old instances of containers to retain globally. Default is -1, which means there is no global limit. **<-- this one is the big change and I think worth implementing to override the defaults. In a past life I've seen production nodes with 3000+ dead containers.**

Reply:
Originally i've set the high TH to 80, removed it on @mj's advice
minimum-container-ttl-duration - set to 5m, so there will be more trace for investigation purposes

Even the logs suggest to increase the default numbers in case of maximum-dead-containers*, and these setting are the same as OSD, for better product alignment.

OSD also has a maximum-dead-containers=2500 set, so I'll add that too

Reply:
We're currently setting `maximum-dead-containers` in this PR to 50, I suspect 200 or 250 might be a good number. (2500 is pathologically large.)

Also, while you're making changes, for some reason there's a mix of single and double quotes around the numbers, might wanna standardize :)

Reply:
Yeah, i ctrl+c, ctrl+v 'd it from the osd files, not sure why it is not consistent there